pkg/helm/handlers/handlers.go

```diff
@@ -43,7 +43,7 @@ func (h *HelmHandlers) HandleHelmRenderManifests(user *auth.User, w http.Respons
 	conf := h.GetActionConfigurations(h.ApiServerHost, req.Namespace, user.Token, &h.Transport)
 	resp, err := h.RenderManifests(req.Name, req.ChartUrl, req.Values, conf)
 	if err != nil {
-		serverutils.SendResponse(w, http.StatusBadGateway, serverutils.ApiError{fmt.Sprintf("Failed to render manifests: %v", err)})
+		serverutils.SendResponse(w, http.StatusBadRequest, serverutils.ApiError{fmt.Sprintf("Failed to render manifests: %v", err)})
```

Comment:
why are we changing contract here? The ticket is about fixing a bug, we should not change the contract. If really needed, we should open another ticket for that.